app/src/main/java/com/aplication/carsales/main_module/view_model/MainViewModel.kt

```diff
@@ -28,9 +30,12 @@ class MainViewModel: ViewModel() {
                 val  resultServer = repository.getCovidDataFromDate(date)
                 result.value = resultServer
                 dateSelected.value = date.split("-").reversed().joinToString("-")
-            } catch (e: Exception) {
+            } catch (e: UnknownHostException){
+                snackBarMsg.value = R.string.unknown_host_error
+            }catch (e: Exception) {
```

Comment:
Remember to format the code before sending it, you can do that with Cmd + Option + L

Extra, about the LiveDatas, you could try making a class that has all the states you need(Like Loading, Empty, Error and Data), and instead of using multiple LiveDatas you can use that one that uses that class.
Not required for this PR but you can take that into consideration